### DISCUSSION TITLE
List favorites in sidebar

### DISCUSSION THREAD

**Discussion by frankaellen** [owncloud/web#5303] [Ideas]
Hey, 

It would be super nice to see all my favorites directly in the sidebar so I as a user can access them quickly and not by first clicking on the general Favorite item.

This could be an option in the user settings maybe and not activated by default? So users can choose to display favorites in the sidebar. Got inspired by the sidebar in the Mac Finder where users can choose which folders should be displayed additionally in the sidebar. 

All the best,
Franka

<img width="843" alt="Screenshot 2021-01-14 at 10 20 19" src="https://user-images.githubusercontent.com/4892549/104570960-d2455c00-5652-11eb-93b8-5f1172c7ab63.png">

**Comment by pascalwengerter** [comment]
@frankaellen should we close this since the sidebars are being redesigned anyways?
